src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -253,16 +258,126 @@ describe('Test suite for mappings on existing concepts', () => {
   });
 
   it('should call removeMappingRow function', () => {
-    const event = { target: { tabIndex: 1 } };
+    const url = '9999';
     const instance = wrapper.find('EditConcept').instance();
     instance.addMappingRow();
+    instance.state.mappings[1] = {
+      map_type: 'Same as',
+      source: INTERNAL_MAPPING_DEFAULT_SOURCE,
+      to_concept_code: null,
+      to_concept_name: null,
+      id: 3,
+      to_source_url: null,
+      isNew: true,
+      retired: false,
+      url: '9999',
+    };
+    expect(instance.state.mappings.filter(_ => _ != null).length).toEqual(2);
+    instance.removeMappingRow(url);
+    expect(instance.state.mappings[1].retired).toEqual(true);
+  });
+
+  it('should call removeUnsavedMappingRow function', () => {
+    const url = '9999';
+    const instance = wrapper.find('EditConcept').instance();
+    instance.state.mappings[1] = {
+      map_type: 'Same as',
+      source: INTERNAL_MAPPING_DEFAULT_SOURCE,
+      to_concept_code: null,
+      to_concept_name: null,
+      id: 3,
+      to_source_url: null,
+      isNew: true,
+      retired: false,
+      url: '9999',
+    };
     expect(instance.state.mappings.filter(_ => _ != null).length).toEqual(2);
-    instance.removeMappingRow(event);
-    expect(instance.state.mappings.filter(_ => _ != null).length).toEqual(1);
+    instance.removeUnsavedMappingRow(url);
+    expect(instance.state.mappings[1].retired).toEqual(true);
+  });
+
+  it('should call removeConceptMappingAction function', () => {
```

Comment:
Are you testing for removeMappingRow or removeConceptMappingAction?

Reply:
It is testing for removeMappingRow